manifests/init.pp

```diff
@@ -31,6 +31,15 @@
 # [*jenkins_url*]
 # The full url (including port) to the jenkins instance.
 #
+# [*install_from_git*]
+# Boolean, defaulst to true. This will install JJB itself from git, but system pkgs will satisfy dependencies
+#
+# [*install_from_pip*]
+# Boolean, defaults to false. Install JJB from pypi using pip
```

Comment:
I think this is inaccurate - in order to persist/retain the current behaviour, we should ensure that the default installation method is via `pip`, not `git` - `pkg` and `git` should be secondary installation methods.